changelogs/versioning.md

```diff
@@ -1,9 +1,9 @@
 # Semantic Versioning

```

Comment:
Shouldn't this live in the `docs` folder?  It doesn't make sense to me in the `changelogs` folder.

Reply:
right, i was skeptical about this too. I created a `.docs` at the root level to host shared docs like this one. is that a dumb idea?